Source/WebKit/NetworkProcess/NetworkProcess.messages.in

```diff
@@ -215,6 +215,9 @@ messages -> NetworkProcess LegacyReceiver {
     RemoveWebPageNetworkParameters(PAL::SessionID sessionID, WebKit::WebPageProxyIdentifier pageID)
     CountNonDefaultSessionSets(PAL::SessionID sessionID) -> (size_t count)
 
+    AllowFileAccessFromWebProcess(WebCore::ProcessIdentifier processIdentifier, String url) -> ()
+    AllowFilesAccessFromWebProcess(WebCore::ProcessIdentifier processIdentifier, Vector<String> urls) -> ()
```

Comment:
Can we make it so there is only one message, and just send a vector of 1 string for the single case?